<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EepromPDAF2D</class>
 <widget class="QWidget" name="EepromPDAF2D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(114, 159, 207);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 28pt &quot;Microsoft YaHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>eeprom PDAF2D</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>162</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>DCCQFormat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_DCCQFormat"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>371</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_14">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>endianness</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_endianness"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>mapWidth</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_mapWidth"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>gainWidth</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_gainWidth"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>leftGainMap</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_leftGainMap"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>conversion Count</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_conversion_count"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>DCCMapWidth</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_DCCMapWidth"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>version</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_version"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>mapHeight</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_mapHeight"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>gainHeight</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_gainHeight"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>rightGainMap</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_rightGainMap"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>conversionCoefficient</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_conversionCoefficient"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>DCCMapHeight</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_DCCMapHeight"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
